src/models/console/consoleRequest.ts

```diff
@@ -12,13 +12,15 @@ export class ConsoleRequest {
     public readonly headers: ko.ObservableArray<ConsoleHeader>;
     public readonly meaningfulHeaders: ko.Computed<ConsoleHeader[]>;
     public readonly representations: ConsoleRepresentation[];
+    public readonly selectedRepresentation: ko.Observable<ConsoleRepresentation>;
     public readonly description: string;
     public readonly body: ko.Observable<string>;
+    public readonly isBodyEdited: ko.Observable<boolean>
     public readonly hasBody: boolean;
     public readonly binary: ko.Observable<File>;
     public readonly bodyFormat: ko.Observable<RequestBodyType>;
     public readonly bodyDataItems: ko.ObservableArray<FormDataItem>;
-    public readonly representationContentType: string;
+    public readonly representationContentType: ko.Observable<string>;
```

Comment:
`public readonly representationContentType: ko.Observable<string>` - this one is not used in the template, and you don't subscribe for its changes, is there a reason making it `ko.Observable`?

Reply:
yes, but at line 44 I update it in a ko subscription, and if don`t define it as observable it doesn`t compile and says "it is a read-only field you can`t change it"

Reply:
just remove `readonly` then :)

Reply:
ah just catch it :)